pkg/render/intrusion_detection_test.go

```diff
@@ -281,33 +326,25 @@ var _ = Describe("Intrusion Detection rendering tests", func() {
 	})
 
 	It("should apply controlPlaneNodeSelector correctly", func() {
-		component := render.IntrusionDetection(nil, nil,
-			&corev1.Secret{ObjectMeta: metav1.ObjectMeta{Name: render.TigeraKibanaCertSecret}},
-			&operatorv1.InstallationSpec{
-				ControlPlaneNodeSelector: map[string]string{"foo": "bar"},
-			},
-			&relasticsearch.ClusterConfig{}, nil, false, dns.DefaultClusterDomain, render.ElasticsearchLicenseTypeUnknown, notManagedCluster,
-			false,
-			nil,
-		)
+		component := render.IntrusionDetection(nil, nil, nil, &corev1.Secret{ObjectMeta: metav1.ObjectMeta{Name: render.TigeraKibanaCertSecret}}, &operatorv1.InstallationSpec{
+			ControlPlaneNodeSelector: map[string]string{"foo": "bar"},
+		}, &relasticsearch.ClusterConfig{}, nil, false, dns.DefaultClusterDomain, render.ElasticsearchLicenseTypeUnknown, notManagedCluster, false, nil, false, nil, nil, nil)
 		resources, _ := component.Objects()
 		idc := rtest.GetResource(resources, "intrusion-detection-controller", render.IntrusionDetectionNamespace, "", "v1", "Deployment").(*apps.Deployment)
 		job := rtest.GetResource(resources, render.IntrusionDetectionInstallerJobName, render.IntrusionDetectionNamespace, "batch", "v1", "Job").(*batchv1.Job)
 		Expect(idc.Spec.Template.Spec.NodeSelector).To(Equal(map[string]string{"foo": "bar"}))
```

Comment:
Does this apply to the daemon set in any way ? Do you mind adding some checks to state that is does not ?